### ISSUE TITLE
Ditch EventEmitter API

### ISSUE THREAD

**Issue by adborden** [goodeggs/librato-node#33]
We don't really document it, but librato-node extends EventEmitter and [emits errors](https://github.com/goodeggs/librato-node/blob/661c1b2b7de55c824b4c5d34d6951d5649730b03/src/librato.coffee#L43) in addition to calling the callback.

This is a bit strange because:
1. It's not documented.
2. You have to handle the error twice, or at least setup two handlers.

There is only one place where we're actually using emit. Should we just drop the EventEmitter API?

**Comment by kevinburkeshyp** [comment]
I'm confused by "handle the error twice", at least if all you are doing is using start, stop and increment() etc, flushing happens in the background and you're not handling it.

I'd prefer if the worker didn't crash when Librato errored, or for the `.on('error')` procedure to be highlighted in the README in big red letters.

**Comment by adborden** [comment]
Good point. I was thinking in the manual flush case, you get the error from the callback *and* the emitter.

If we decide to keep it, I agree that we should document it.

**Comment by icodeforlove** [comment]
This one just bit me, there really needs to be documentation on this. Do i understand this correctly, if i have a 

```
liberato.on('error', function() {});
```

I will no longer have uncaught errors when there are network issues?

**Comment by adborden** [comment]
Yeah, that's right. I just added this to the readme #46

**Comment by bobzoller** [comment]
thanks @adborden!

**Comment by timborden** [comment]
Bit me too....thanks @adborden for adding to the docs
